core/src/plugins/filed/python/vmware/BareosFdPluginVMware.py

```diff
@@ -2668,6 +2681,8 @@ def transform(self, target_datastore_name=None):
         ]
         config_spec.migrateEncryption = self.config_info["migrateEncryption"]
         config_spec.name = self.config_info["name"]
+        if target_vm_name:
```

Comment:
Shouldn't we add a line into documentation https://docs.bareos.org/bareos-22/TasksAndConcepts/Plugins.html#description-of-all-plugin-options if vmname is present during restore the vm will be restored with the new name ?
Seems obvious but ...